game/static/game/css/landing.css

```
@@ -309,37 +309,89 @@ body {
+
+.footer-col p {
+    color: var(--text-secondary);
+    font-size: 0.95rem;
+    line-height: 1.6;
+}
+
+.footer-col h4 {
+    color: var(--text-title);
+    font-size: 1.05rem;
+    font-weight: 700;
+    font-family: 'Cinzel', serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)